tests/model_hub_tests/torch_tests/test_torchvision_models.py

```diff
@@ -0,0 +1,103 @@
+
+    frames, _, _ = read_video(str(video_path), output_format="TCHW")
+    return frames
+
+
+def prepare_frames_for_raft(name, frames1, frames2):
+    w = torch.hub.load("pytorch/vision", "get_model_weights",
+                       name=name).DEFAULT
+    img1_batch = torch.stack(frames1)
+    img2_batch = torch.stack(frames2)
+    img1_batch = F.resize(img1_batch, size=[520, 960], antialias=False)
+    img2_batch = F.resize(img2_batch, size=[520, 960], antialias=False)
+    img1_batch, img2_batch = w.transforms()(img1_batch, img2_batch)
+    return (img1_batch, img2_batch)
+
+
+class TestTorchHubConvertModel(TestConvertModel):
+    def load_model(self, model_name, model_link):
+        m = torch.hub.load("pytorch/vision", model_name, weights='DEFAULT')
+        m.eval()
+        if model_name == "s3d" or any([m in model_name for m in ["swin3d", "r3d_18", "mc3_18", "r2plus1d_18"]]):
+            self.example = (torch.randn([1, 3, 224, 224, 224]),)
+            self.inputs = (torch.randn([1, 3, 224, 224, 224]),)
+        elif "mvit" in model_name:
+            # 16 frames from video
+            self.example = (torch.randn(1, 3, 16, 224, 224),)
+            self.inputs = (torch.randn(1, 3, 16, 224, 224),)
+        elif "raft" in model_name:
+            frames = get_video()
+            self.example = prepare_frames_for_raft(model_name,
+                                                   [frames[100], frames[150]],
+                                                   [frames[101], frames[151]])
+            self.inputs = prepare_frames_for_raft(model_name,
+                                                  [frames[75], frames[125]],
+                                                  [frames[76], frames[126]])
+        else:
+            self.example = (torch.randn(1, 3, 224, 224),)
+            self.inputs = (torch.randn(1, 3, 224, 224),)
+        return m
+
+    def get_inputs_info(self, model_obj):
+        return None
+
+    def prepare_inputs(self, inputs_info):
+        return [i.numpy() for i in self.inputs]
+
+    def convert_model(self, model_obj):
+        ov_model = convert_model(model_obj, example_input=self.example)
+        return ov_model
+
+    def infer_fw_model(self, model_obj, inputs):
+        fw_outputs = model_obj(*[torch.from_numpy(i) for i in inputs])
+        if isinstance(fw_outputs, dict):
+            for k in fw_outputs.keys():
+                fw_outputs[k] = fw_outputs[k].numpy(force=True)
+        elif isinstance(fw_outputs, (list, tuple)):
+            fw_outputs = [o.numpy(force=True) for o in fw_outputs]
+        else:
+            fw_outputs = [fw_outputs.numpy(force=True)]
+        return fw_outputs
+
+    @pytest.mark.parametrize("model_name", ["efficientnet_b7", "raft_small", "swin_v2_s"])
+    @pytest.mark.precommit
+    def test_convert_model_precommit(self, model_name, ie_device):
+        self.run(model_name, None, ie_device)
+
+    @pytest.mark.parametrize("model_name", get_all_models())
```

Comment:
please create a file with nightly scope model. I am afraid that `get_all_model()` can issue different number of models from time to time so it is better to have a concrete list and update it manually.

Reply:
why? If some model was removed we will not test it, if model was added we will test new model. Having list of models is not easy to handle